app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by!(username: params.fetch(:id))
```

Comment:
change to `params.fetch(:username)`

Reply:
Hi Jared, Thank you so much for taking a look! I really appreciate your feedback